.pipelines/prchecks/PackageBuildPRCheck.yml

```diff
@@ -36,6 +34,7 @@ resources:
 
 variables:
   - group: "Agent pools (DEV)"
+  - group: "Raw toolchain info"
```

Comment:
Why are there two group entries here?

Reply:
Because we're using two groups. One has the info about the dev build agents and the other about the raw toolchain. There's a comment at the top of the YAML with a bit more details regarding the usage of these two groups.